osuosl/climesync
# Scripting mode added

fixes issue #39 
Also addresses #38

## Changes in this PR.


- [X] Scripting mode added to Climesync

## Testing this PR.


1. Run `nosetests`
2. Run `./climesync.py --help` to see all the command line options
3. Use those command line options to interact with TimeSync

@osuosl/devs
